api-server-rest/build.rs

```diff
@@ -0,0 +1,102 @@
+use ttrpc_codegen::{Codegen, Customize, ProtobufCustomize};
```

Comment:
do you need SPDX-License-Identifier here?

Reply:
Thanks @Xynnn007 , good catch, fixed now：）